<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageMode</class>
 <widget class="QDialog" name="ImageMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="run_pushButton">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>310</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>451</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="image_csv_button">
        <property name="text">
         <string>Choose ImageInfo CSV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="image_csv_label">
        <property name="text">
         <string>NA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="map_points_button">
        <property name="text">
         <string>Choose Map Points </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="map_points_label">
        <property name="text">
         <string>NA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="image_dir_button">
        <property name="text">
         <string>Choose Image Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="image_dir_label">
        <property name="text">
         <string>NA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="cam_settings_button">
        <property name="text">
         <string>Choose Camera Setting</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cam_settings_label">
        <property name="text">
         <string>NA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>image_dir_button</zorder>
  <zorder>image_csv_button</zorder>
  <zorder>image_dir_label</zorder>
  <zorder>image_csv_label</zorder>
  <zorder>run_pushButton</zorder>
  <zorder>cam_settings_button</zorder>
  <zorder></zorder>
  <zorder>map_points_button</zorder>
  <zorder>map_points_label</zorder>
  <zorder>cam_settings_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
